requirements-dev.txt

```diff
@@ -236,4 +236,3 @@ wheel==0.40.0
 # The following packages are considered to be unsafe in a requirements file:
 # pip
 # setuptools
-setuptools>=65.5.1 # not directly required, pinned by Snyk to avoid a vulnerability
```

Comment:
If you're wondering why Snyk is angry.

Reply:
ahh, I think it would make sense then that it would be in the requirements-dev.in so it won't get wiped when upgrading packages with `pip-compile`

Reply:
An excellent idea!

Reply:
The idea went down to drain since you cannot add setuptools to .in file :(